<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lab4</class>
 <widget class="QMainWindow" name="lab4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1549</width>
    <height>877</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="pic">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1221</width>
      <height>811</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>760</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="w_loc">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>210</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1430</x>
      <y>760</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>恢复</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>100</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>窗宽：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="w_wid">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>130</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>180</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>窗位：</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="img_window">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>60</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>灰度窗</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="img_intensify">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>290</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>图像增强</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="img_save">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>360</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>保存图像</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="save_name">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>430</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>400</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>图像名称</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1549</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="actionraw"/>
    <addaction name="actionother"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionraw">
   <property name="text">
    <string>raw</string>
   </property>
  </action>
  <action name="actionother">
   <property name="text">
    <string>other</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
